.ci/pipeline_definitions

```diff
@@ -16,28 +18,33 @@ gardener-custom-metrics:
         - linux/arm64
         dockerimages:
           gardener-custom-metrics:
-            registry: 'gcr-readwrite'
-            image: 'eu.gcr.io/sap-se-gcp-scp-k8s/gardener/gardener-custom-metrics' # 'eu.gcr.io/gardener-project/gardener/gardener-custom-metrics'
+            image: 'europe-docker.pkg.dev/gardener-project/snapshots/gardener/gardener-custom-metrics'
             dockerfile: 'Dockerfile'
             target: gardener-custom-metrics
   jobs:
     head-update:
       traits:
-        component_descriptor: ~
+        component_descriptor:
+          ocm_repository_mappings:
+            - repository: europe-docker.pkg.dev/gardener-project/releases
         draft_release: ~
         options:
           public_build_logs: true
     pull-request:
       traits:
         pull-request: ~
-        component_descriptor: ~
+        component_descriptor:
+          ocm_repository_mappings:
+            - repository: europe-docker.pkg.dev/gardener-project/releases
```

Comment:
this is correct, if you always reference final versions from other components (.ci/component_descriptor callback). if you want to be able to also reference snapshot-versions, you should add `/snapshots` ocm repository as second mapping entry.